wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
@mikekaganski When parallel processing is in use, the module is loaded independently on each thread. So each thread will have it's own buffer.
```Julia
$ julia -p 2
julia> using CoolProp
julia> fetch(@spawnat 1 CoolProp.message_buffer)
255-element Array{UInt8,1}:
 0x11
 0x27
 0xcf
 0xfc
 0x05
    ⋮
 0x00
 0x00
 0x00
 0x00

julia> fetch(@spawnat 2 CoolProp.message_buffer)
255-element Array{UInt8,1}:
 0x00
 0x00
 0x00
 0x00
 0x00
    ⋮
 0x00
 0x00
 0x00
 0x00
```
The two buffer have different content depending on the thread. If I modify the one of the 1rst thread, only that one is changed.

Thanks for your advices